examples/asset-upload/package.json

```diff
@@ -32,7 +32,7 @@
     },
     "dependencies": {
         "@frontify/app-bridge": "^3.3.1",
-        "@frontify/fondue": "12.0.0-beta.404",
+        "@frontify/fondue": "12.0.0-beta.406",
         "@frontify/guideline-blocks-settings": "^0.33.1",
```

Comment:
```suggestion
        "@frontify/guideline-blocks-settings": "^0.34.0",
```

Reply:
Thanks fixed. And added fondue styles to all blocks.